internal/provider/resource_ko_build.go

```diff
@@ -311,10 +314,15 @@ func resourceKoBuildRead(ctx context.Context, d *schema.ResourceData, meta inter
 		}
 		return diag.Errorf("[id=%s] read doBuild: %v", d.Id(), err)
 	}
+	dig, err := res.Digest()
+	if err != nil {
+		return diag.Errorf("[id=%s] read digest: %v", d.Id(), err)
+	}
+	digstr := dig.String()
```

Comment:
I think you want this to be `.Digest(dig.String())` on an image ref?

as-is, this is just the `sha256:deadbeef` part, which I think it incorrect

Reply:
Done!